clang-tools-extra/docs/clang-tidy/index.rst

```diff
@@ -292,6 +292,18 @@ An overview of all the command-line options:
     Checks                       - Same as '--checks'. Additionally, the list of
                                    globs can be specified as a list instead of a
                                    string.
+    ClangQueryChecks             - List of key-value pairs. Key specifies a name
```

Comment:
this should be documented in separate section, thats not a good place for example